drivers/volume/portworx/portworx.go

```diff
@@ -215,7 +215,7 @@ func (d *portworx) RecoverDriver(n node.Node) error {
 		return nil, true, fmt.Errorf("Node %v is not in Maintenance mode", n.Name)
 	}
 
-	if _, err := task.DoRetryWithTimeout(t, 1*time.Minute, 10*time.Second); err != nil {
+	if _, err := task.DoRetryWithTimeout(t, 2*time.Minute, 10*time.Second); err != nil {
```

Comment:
Since you are at it, can you move timeouts in this file to  a single `const` section at the top. Basically something like https://github.com/portworx/talisman/blob/master/pkg/cluster/px/px.go#L86

Reply:
👍 
Done!